### PR TITLE
Make kubetestPath robust to cd

### PR DESCRIPTION
**Pull request by fejta**
We cannot just use `os.Args[0]` because `--extract=E` changes `os.Getwd()` join these two together before we change directories.

Am testing this locally on an upgrade job, also expanding the canary job to cover this scenario

### CODE REVIEW

**Comment by krzyzacy** [comment]
ah whoops I was giving @krousey bad ideas here sorry :cold_sweat: 

this is the right way to go :-)

/lgtm

File: kubetest/main.go [hunk 1]
```diff
@@ -34,12 +34,22 @@ import (
...
 var (
 	interrupt             = time.NewTimer(time.Duration(0)) // interrupt testing at this time.
+	kubetestPath          = "need-to-call-setKubetestPath()"
 	terminate             = time.NewTimer(time.Duration(0)) // terminate testing at this time.
 	verbose               = false
 	timeout               = time.Duration(0)
 	deprecatedVersionSkew = flag.Bool("check_version_skew", true, "Verify client and server versions match")
 )
 
+func setKubetestPath() error {
```
Comment (krousey) [inline kubetest/main.go]: maybe just make this an init func?

Comment (fejta) [inline kubetest/main.go]: Done

**Pull request merged**